### PR TITLE
Create OpenItemsonVisionLaunch.vb

### PR DESCRIPTION
**Pull request by sybethanlevy**
Input please re lines 12-14 and device settings; if not possible will dump the idea of course.

### CODE REVIEW

File: BreakpointSamples/Modifying the Standard Workflow of Vision/OpenItemsonVisionLaunch.vb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    'Prerequisities: A custom grid created called "Outstanding Orders"
+    'Breakpoints: VisionCompanyLaunched
+	
+	
+		Public Sub Invoke(ByVal e As Sybiz.Vision.Platform.Admin.Breakpoints.BreakpointEventArgs) 'Do not remove - SYBIZ
+
+			If Sybiz.Vision.Platform.Security.Principal.CurrentPrincipal.IsInRole("POS") = True Then
+					
+				'Since it's POS; is it possible to 'force' KioskMode on? This seems like the way to do it, but does not work (due to Object Reference not Set...); guessing it's intentional but can't hurt to ask!
+				Dim ds As Sybiz.Vision.Platform.Admin.DeviceSettings
```
Comment (Marjon80) [inline BreakpointSamples/Modifying the Standard Workflow of Vision/OpenItemsonVisionLaunch.vb]: If (Sybiz.Vision.Platform.Admin.DeviceSettings.CanAccess()) Then
Dim ds As Sybiz.Vision.Platform.Admin.DeviceSettings =  Sybiz.Vision.Platform.Admin.DeviceSettings.GetObject()

File: BreakpointSamples/Modifying the Standard Workflow of Vision/OpenItemsonVisionLaunch.vb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+				ds.KioskMode = true
+
```
Comment (Marjon80) [inline BreakpointSamples/Modifying the Standard Workflow of Vision/OpenItemsonVisionLaunch.vb]: ds = ds.Save()

**Comment by sybethanlevy** [comment]
Please refer above commit @Marjon80 - cheers for the pointers in the right direction! [d55d79a](https://github.com/Sybiz/Vision/pull/2/commits/d55d79a6973f6cf0cebb4ed35d6989ac5f052faf)

**Comment by sybethanlevy** [comment]
Actually please see updated [466f30d](https://github.com/Sybiz/Vision/pull/2/commits/466f30db09a5d93861c9fec1143f8774f7b95282), now featuring one less silly mistake.

**Pull request merged**